### DISCUSSION TITLE
dall'analisi del log risulta un errore relativo a ZigBee2Mqtt

### DISCUSSION THREAD

**Discussion by PiuCurioso** [indomus/forum#1574] [ZigBee, Z-Wave, Thread, Matter]
Carissimi, ciao vi scrivo per un consiglio analizzando questo log ( che allego) trovo degli errori relativi a ZigBee2Mqtt potreste darmi una mano per capire di che si tratta ed eventualemtre come risolvere? grazie mille un caro saluto Buona Domenica

![Errore 13 03 2022](https://user-images.githubusercontent.com/55890118/158071767-d08711e2-b38a-4eaf-b5d7-f1d19a060a92.jpg)
![Errore 13 03 2022 pag 2](https://user-images.githubusercontent.com/55890118/158071783-176cabc9-89b6-41bf-b964-98027bcfcefb.jpg)

**Comment by snakuzzo** [comment]
Ti chiediamo cortesemente di postare i log in maniera "decente". Non con gli screenshot e magari solo quelli riguardanti zigbee2mqtt...ovviamente utilizzando il markdown.
Grazie.

**Reply by PiuCurioso** [reply]
Ciao Snakuzzo scusami, per cui se non ho capito male non devo allegare immagini ,ma devo copiare l'errore all'interno di un campo che si attiva utilizzando il markdown? Cioè per attivare il markdown quale pulsante si usa?Grazie e scusami ti auguro una buona giornata

**Reply by snakuzzo** [reply]
leggi [QUI](https://github.com/indomus/forum/discussions/1311)

**Reply by PiuCurioso** [reply]
Ok grazie Diego un caro saluto e scusami
